trustpoint/templates/home/notifications-tab.html

```diff
@@ -130,16 +138,31 @@ <h1>{% trans 'Notifications' %}</h1>
                     <td>{{ notification.get_notification_source_display }}</td>
                     <td>{{ notification.short_translated }}</td>
                     <td>{{ notification.created }}</td>
-                    <td><a href="{% url 'home:notification_details' notification.id %}" class="btn btn-primary tp-table-btn">
+                    <td><a href="{% url 'home:notification_details' notification.id %}"
+                           class="btn btn-primary tp-table-btn">
                         {% trans 'Details' %}
                     </a></td>
                 </tr>
-                {% empty %}
-                <tr><td colspan="5" class="middle">{% trans 'No notifications available.' %}</td></tr>
-                {% endfor %}
+            {% empty %}
+                <tr>
+                    <td colspan="5" class="middle">{% trans 'No notifications available.' %}</td>
+                </tr>
+            {% endfor %}
             </tbody>
         </table>
         {% include 'trustpoint/pagination.html' %}
+        <div class="row py-3 px-0">
+            <div class="d-flex justify-content-between align-items-center">
+                <a href="{% url 'notifications:execute_notifications' %}" class="btn btn-primary">
+                    <svg class="icon me-2" width="16" height="16" fill="currentColor">
+                        <use xlink:href="{% static 'img/icons.svg' %}#icon-refresh"></use>
+                    </svg>
+                    {% trans 'Execute Notifications' %}
```

Comment:
Maybe we should call this `Refresh notifications`? I don't think the average user would understand the meaning of `Execute` here and could think it means that the system could auto-fix issues or similar...